lib/plug/csrf_protection.ex

```diff
@@ -46,7 +46,11 @@ defmodule Plug.CSRFProtection do
   ## Disabling
 
   You may disable this plug by doing
-  `Plug.Conn.put_private(:plug_skip_csrf_protection, true)`.
+  `Plug.Conn.put_private(:plug_skip_csrf_protection, true)`. This was made
+  available for disabling `CSRFProtection` in tests and not for dynamically
```

Comment:
`CSRFProtection` is not a valid reference. So we should either go with "CSRF protection" or `Plug.CSRFProtection`. :)